md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
```

Comment:
There is a substantial difference between being grumpy and being grumpy *at other people* who are doing their best to help. The former is fine. The latter must be prohibited – it's contagious and destructive, and IMO is better solved by just taking some time away and doing what you need to do to return to normal.